docs/examples/example.ipynb

```diff
@@ -491,7 +491,7 @@
     "{\n",
     "  \"components\": [\n",
     "    {\n",
-    "      \"model\": \"acme:wifi_sensor:linux\",\n",
+    "      \"model\": \"viam:sensor:linux-wifi\",\n",
```

Comment:
this needs to match the Model above

Reply:
what Model does this refer to?

Reply:
Okay, I changed all instances of `wifi-sensor` to `linux-wifi` to be consistent with our example doc [here](https://github.com/viam-labs/wifi-sensor/blob/main/meta.json)

Reply:
lets change the model above then